auth/util_test.go

```diff
@@ -33,14 +33,14 @@ func (s *UtilSuite) Test_getID() {
 
 func (s *UtilSuite) Test_getToken() {
 	ctx, _ := gin.CreateTestContext(httptest.NewRecorder())
-	RegisterAuthentication(ctx, nil, 1, "asdasda")
+	RegisterAuthentication(ctx, nil, "asdasda")
```

Comment:
nil, as user isn't allowed anymore, there are a lot occurrences of this in tests.

Reply:
Sorry could you please explain this, which should be nil?